components/Form/Details/index.module.scss

```diff
@@ -8,4 +10,8 @@
 
 .lbh-details__text {
   border-left: 0;
+
+  @media (max-width: #{$screen-sm-min}) {
+    word-break: break-word;
+  }
```

Comment:
Not sure how long these links could get, but is it worth this on all screen resolutions do you think?

Reply:
Hmm

Reply:
I'm gonna try a supermassive link